### ISSUE TITLE
Dumping of statistical results

### ISSUE THREAD

**Issue by marinang** [scikit-hep/hepstats#19]
After a chat yesterday at CERN with @mayou36 and Carsten, a RooFit developer, we concluded that storing the results of `hepstats` is crucial as they can reused, for plotting for instance, without re-running CPU-intensive calculations done in `hepstats`.

The idea is to store in a dictionary:

- best fit result
- likelihood scan
- significance 
- upper limits / intervals
- null and alternative hypothesis 
- parameter ranking (not implement yet)

This dictionary can be easily serialised into yml or json file.

**Comment by eduardo-rodrigues** [comment]
Brilliant, really nice to see these ideas moving forward (a lot of this has been discussed among the community in recent times).

cc @benkrikler - folks, I would like to organise a PyHEP WG meeting early next year as a follow-up to the fitting topical meeting we had. Seems to me that fitting tools and connections to statistics tools are a hot and important topic. What would be the timescale for you folks to be available for a dedicated meeting? I would imagine a couple of short presentations and a few slides backing up an open discussion. We can continue this discussion offline as it's far more general than this repo :-D.

**Comment by mayou36** [comment]
Yes, that sounds good, I am mostly available in the new year. 

Additionally, we also discussed the storage of the likelihood, most importantly of the fitting model. Since this is a larger topic, maybe overlapping with preservation (?), do you know by chance people that are also working on that?

**Comment by eduardo-rodrigues** [comment]
Good point, @mayou36. I do know at least 2 relevant people, namely Lukas and Matthew from the pyhf team. I will throw an email soon to start organising a meeting … Not to be done here on GitHub.

**Comment by mayou36** [comment]
Thanks, @eduardo-rodrigues those I know and we're already in contact. In fact, it's them, Carsten (from RooFit) and me who are mostly interested in this.
But if you know others (more from analysis preservation?), let me know

**Comment by eduardo-rodrigues** [comment]
I thought you would … I think we will get more names is the preparations of a meeting, especially that I will include the HSF analysis WG in there too, since the link is very relevant in this instance ...
